ConsumerApi/ConsumerApi.csproj

```diff
@@ -33,6 +33,7 @@
 		<PackageReference Include="Serilog" Version="3.0.1" />
 		<PackageReference Include="Serilog.AspNetCore" Version="7.0.0" />
 		<PackageReference Include="Serilog.Enrichers.Environment" Version="2.2.0" />
+		<PackageReference Include="Serilog.Enrichers.CorrelationId" Version="3.0.1" />
```

Comment:
I hadn't found that before. It seems the offiial package has ~400x less downloads than the initial one, but the functionality is the same (tested), so let's go with the official one :)

Reply:
Seems like the official one is rather new (first commit in July 2022). 

But what bothers me is that the last commit was in July 2022 as well. So no updates of any dependencies since then. So probably it's not a good idea to use this. :-|